src/class/incomer.class.ts

```diff
@@ -746,6 +761,7 @@ export class Incomer <
 
     const callbackResult = await this.eventCallback({ ...event, eventTransactionId } as unknown as CallBackEventMessage<T>);
 
+    let reason = callbackResult.val;
     if (callbackResult && callbackResult.ok) {
```

Comment:
`callbackResult &&` condition is necessary ?
Because on the above line, if `callbackResult` is undefined or null, there will be a `TypeError`.